docs/migration-guides/0.8-0.9.md

```diff
@@ -0,0 +1,13 @@
+# Migration Guide 0.8 - 0.9
+
+## Take advantage to the new polyfilling
+
+We updated the `babel-preset-kyt-core` and `babel-preset-kyt-react` presets with `babel-preset-env`. Before 0.9, kyt automatically included `babel-polyfill` into your bundle. With the recent changes, you'll have to manually install `babel-polyfill` into your package.json `dependencies` (not `devDependencies`), and `import 'babel-polyfill;'` at the top of your `src/client/index.js` and `src/server/index.js` file. This will optimize your server build by targeting the current version of Node and by targeting a limited set of browsers using a [browserlist configuration](https://github.com/ai/browserslist) on the client build. You can read more about extending the `babel-preset-env` settings via the [`envOptions` in `babel-preset-kyt-core`](/packages/babel-preset-kyt-core/README.md#options).
+
+## Static asset handling updates
+
+In 0.9, we added better support for static assets, including new rules and fingerprinting. You'll find more documentation in the [conventions doc](/docs/conventions.md#static-asset-handling-assets-manifest-cdn).
+
+## Linting against the server directory
+
+A new rule in `eslint-config-kyt` is configured to restrict server file imports from outside of the `src/server` directory. The change you'll need to make in your project is to add [this .eslintrc.js](https://github.com/NYTimes/kyt/blob/7b1a019b1e0996c94df2a5da48b2062c7113ce02/packages/kyt-starter-universal/starter-src/src/server/.eslintrc.js) to your `src/server` directory. Going forward, starter kyts will have the proper configuration for this to work.
```

Comment:
Instead of `Going forward, starter kyts will have the proper configuration for this to work.` Say `The current starter-kyt version includes this configuration.`